### PR TITLE
RM-925 RN 2.19.0

### PR DESCRIPTION
**Pull request by PaulJacobs-percona**
Preview
https://60d5dc5b48dcdc0008ccea51--pmm-doc.netlify.app/release-notes/2.19.0.html

### CODE REVIEW

**Comment by PaulJacobs-percona** [comment]
https://60ca087c8feb6600079ecb0d--pmm-doc.netlify.app/release-notes/2.19.0.html

File: docs/release-notes/2.19.0.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+## Release Highlights
+
+- Backup Management can now be turned on from the UI. Go to *Settings-->Advanced settings*, and you will see it in the *Technical Preview* section. We also added support of MongoDB services on-demand backup and restore. For now, it only supports ReplicaSet on S3-compatible storage.
+
+- Dashboards improvements
+
+    - There are several community-driven improvements to ProxySQL data collection, with new dashboards to expose such metrics like: *Queries Latency histograms* and `SHUNNED_REPLICATION_LAG` state.
+
+    - Fixes for Amazon Aurora service detection on dashboards (LINK), MongoDB ReplicaSet Summary, and other MongoDB memory-related panels.
```
Comment (rnovikovP) [inline docs/release-notes/2.19.0.md:31]:
```suggestion
    - Fixes for Amazon Aurora service detection on [the dashboard](https://pmmdemo.percona.com/graph/d/mysql-amazonaurora/mysql-amazon-aurora-details?var-service_name=rds-aurora57-instance-1), MongoDB ReplicaSet Summary, and other MongoDB memory-related panels.
```

File: docs/release-notes/2.19.0.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+- Backup Management can now be turned on from the UI. Go to *Settings-->Advanced settings*, and you will see it in the *Technical Preview* section. We also added support of MongoDB services on-demand backup and restore. For now, it only supports ReplicaSet on S3-compatible storage.
+
+- Dashboards improvements
+
+    - There are several community-driven improvements to ProxySQL data collection, with new dashboards to expose such metrics like: *Queries Latency histograms* and `SHUNNED_REPLICATION_LAG` state.
+
+    - Fixes for Amazon Aurora service detection on [the dashboard](https://pmmdemo.percona.com/graph/d/mysql-amazonaurora/mysql-amazon-aurora-details?var-service_name=rds-aurora57-instance-1), MongoDB ReplicaSet Summary, and other MongoDB memory-related panels.
+
+- Improvements to the security of DBaaS secrets by generating passwords for installed operators. We shipped Automated operator installation in provided K8s cluster in PMM 2.18, and here we are improving its security.  (???)
```
Comment (shoffman-percona) [inline docs/release-notes/2.19.0.md:33]:
```suggestion
- Improvements to DBaaS secrets by generating strong passwords for operators. This is an improvement to the Automated Operator Installation released in PMM 2.18, which will greatly enhance security.  
```

File: docs/release-notes/2.19.0.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+## New Features
+
+* [PMM-7639](https://jira.percona.com/browse/PMM-7639): Backup Management: Ability to remove Backup Location even if there are some backup artifacts on it
+* [PMM-7568](https://jira.percona.com/browse/PMM-7568): Backup Management: Simple restore for MongoDB ReplicaSet
+* [PMM-7567](https://jira.percona.com/browse/PMM-7567): Backup Management: Simple backup for MongoDB ReplicaSet
```
Comment (shoffman-percona) [inline docs/release-notes/2.19.0.md:39]:
```suggestion
* [PMM-7567](https://jira.percona.com/browse/PMM-7567): Backup Management: Simple backup for MongoDB ReplicaSet
* [PMM-7568](https://jira.percona.com/browse/PMM-7568): Backup Management: Simple restore for MongoDB ReplicaSet
```

File: docs/release-notes/2.19.0.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+
+
+## Improvements
+* [PMM-8112](https://jira.percona.com/browse/PMM-8112): Dashboards: Collect and present on dashboard Queries Latency histograms from ProxySQL
```
Comment (shoffman-percona) [inline docs/release-notes/2.19.0.md:44]:
```suggestion
* [PMM-8112](https://jira.percona.com/browse/PMM-8112): Dashboards: Collect and present histograms from ProxySQL on Queries Latency dashboard
```

File: docs/release-notes/2.19.0.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+* [PMM-8081](https://jira.percona.com/browse/PMM-8081): Dashboards: Add collection and presentation for `SHUNNED_REPLICATION_LAG` state to `proxysql_connection_pool_status`
+* [PMM-7584](https://jira.percona.com/browse/PMM-7584): Components Upgrade: VictoriaMetrics 1.53.1 to 1.60.0 TBD (link for VM features)
+* [PMM-8001](https://jira.percona.com/browse/PMM-8001): Better error handling when `pg_stat_monitor` has an unsupported version
```
Comment (shoffman-percona) [inline docs/release-notes/2.19.0.md:47]:
```suggestion
* [PMM-8001](https://jira.percona.com/browse/PMM-8001): Better error handling when `pg_stat_monitor` is an unsupported version
```

File: docs/release-notes/2.19.0.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+* [PMM-7659](https://jira.percona.com/browse/PMM-7659): DBaaS: Ability to specify the type of connection for DBaaS cluster during DB Cluster creation
+* [PMM-7828](https://jira.percona.com/browse/PMM-7828): DBaaS: Select Database Type by default if only one operator is installed
+* [PMM-8153](https://jira.percona.com/browse/PMM-8153): Backup Management: Improve UX for Backup lists when Service acceptable for restore is removed
```
Comment (shoffman-percona) [inline docs/release-notes/2.19.0.md:50]: I have no idea what this means?  Does this mean when there is no acceptable restore service/location available?

File: docs/release-notes/2.19.0.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+
+
+
+## Bugs Fixed
+
+* [PMM-7194](https://jira.percona.com/browse/PMM-7194): 'Share with Percona' option doesn't export all panels
```
Comment (shoffman-percona) [inline docs/release-notes/2.19.0.md:56]:
```suggestion
* [PMM-7194](https://jira.percona.com/browse/PMM-7194): 'Share with Percona' option doesn't export data from collapsed panels
```

**Review by shoffman-percona — CHANGES_REQUESTED** [review]
couple suggestions and one I couldn't figure out what was meant...

File: docs/release-notes/2.19.0.md (hunk 6 above)
Comment (PaulJacobs-percona) [reply docs/release-notes/2.19.0.md:50]:
```suggestion
[PMM-8153](https://jira.percona.com/browse/PMM-8153): Backup Management: Disable 'Restore' button for backups whose service has been deleted
```

Comment (PaulJacobs-percona) [reply docs/release-notes/2.19.0.md:50]: @rnovikovP Is this ok?

File: docs/release-notes/2.19.0.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+## New Features
+
+* [PMM-7639](https://jira.percona.com/browse/PMM-7639): Backup Management: Ability to remove Backup Location even if there are some backup artifacts on it
+* [PMM-7567](https://jira.percona.com/browse/PMM-7567): Backup Management: Simple backup for MongoDB ReplicaSet
+* [PMM-7568](https://jira.percona.com/browse/PMM-7568): Backup Management: Simple restore for MongoDB ReplicaSet
+
+
+
+## Improvements
+* [PMM-8112](https://jira.percona.com/browse/PMM-8112): Dashboards: Collect and present histograms from ProxySQL on Queries Latency dashboard
```
Comment (rnovikovP) [inline docs/release-notes/2.19.0.md:44]:
```suggestion
* [PMM-8112](https://jira.percona.com/browse/PMM-8112): Dashboards: Collect and present histograms from ProxySQL on Queries Latency dashboard (Thanks to [foosin](https://github.com/foosinn) for helping with this improvement )
```

File: docs/release-notes/2.19.0.md [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+* [PMM-8081](https://jira.percona.com/browse/PMM-8081): Dashboards: Add collection and presentation for `SHUNNED_REPLICATION_LAG` state to `proxysql_connection_pool_status`
```
Comment (rnovikovP) [inline docs/release-notes/2.19.0.md:45]:
```suggestion
* [PMM-8081](https://jira.percona.com/browse/PMM-8081): Dashboards: Add collection and presentation for `SHUNNED_REPLICATION_LAG` state to `proxysql_connection_pool_status` 
(Thanks to [spaceform](https://github.com/spaceform) for helping with this improvement )
```

File: docs/release-notes/2.19.0.md [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+
+
+
+## Bugs Fixed
+
+* [PMM-7194](https://jira.percona.com/browse/PMM-7194): 'Share with Percona' option doesn't export data from collapsed panels
+* [PMM-7104](https://jira.percona.com/browse/PMM-7104): Slowlog rotation by pmm-agent causing additional unexpected rotation of binary logs (Thanks to Arthur Tokarchuk for reporting this issue)
+* [PMM-8125](https://jira.percona.com/browse/PMM-8125): Error of monitoring GCP based PostgreSQL because of internal database `cloudsqladmin`
+* [PMM-6778](https://jira.percona.com/browse/PMM-6778): Can't specify custom labels during node addition/configuration
+* [PMM-8090](https://jira.percona.com/browse/PMM-8090): Multi-request protection breaks metrics gathering
```
Comment (rnovikovP) [inline docs/release-notes/2.19.0.md:60]:
```suggestion
* [PMM-8090](https://jira.percona.com/browse/PMM-8090): Multi-request protection breaks metrics gathering (Thanks to [Francisco Miguel Biete](https://github.com/fmbiete) for fixing this bug )
```

**Pull request merged**